aws/resource_aws_rds_cluster.go

```diff
@@ -232,6 +232,11 @@ func resourceAwsRDSCluster() *schema.Resource {
 				Optional: true,
 			},
 
+			"source_region": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```

Comment:
I believe we should mark this as `ConflictsWith: []string{"snapshot_identifier"}` because this field will be ignored if the DB is restored, instead of created from scratch.

Reply:
Also it seems it's not updatable, so we should mark it as `ForceNew: true`.